app/src/main/java/com/shevchenkovtwo/homework/AppConstants.kt

```diff
@@ -1,7 +1,7 @@
 package com.shevchenkovtwo.homework
 
 object AppConstants {
```

Comment:
This is not really scalable - you have two screens and already three constants/variables. This file will grow enormously when app will grow.